h/static/scripts/annotator/test/integration/anchoring-test.js

```diff
@@ -99,4 +100,24 @@ describe('anchoring', function () {
     ],
     expectFail: true,
   }]);
+
+  unroll('should silently fail when an annotation does not have a quote selector', function (testCase) {
+    var annotation = testCase.annotation;
+    var anchored = guest.anchor(annotation);
+
+    return promiseUtil.toResult(anchored).then(function (result) {
```

Comment:
This looks fine, except that `promiseUtil.toResult()` is no longer required since we're expecting the promise to resolve.